.github/workflows/pr_title_check.yml

```
@@ -2,7 +2,9 @@ name: Check PR title
 
 on:
   pull_request:
-    branches: ["**"]
+    branches: 
+      - '**'
+      - '!release'
```

I'm assuming this now needs to be 
```suggestion
      - '!release/v*'
```

---

Resolved.